Fix TC_CADMIN_1_5

doublemis1 opened the pull request:
Test Step 4 is failing because the test case is not expecting the timeout error while the commissioning window is already closed (after timeout).

### Testing:

Using nrfconnect  platform based on Matter 1.5 over Thread version

andy31415 commented on src/python_testing/TC_CADMIN_1_5.py:
```diff
@@ -128,7 +128,7 @@ async def test_TC_CADMIN_1_5(self):
 
         self.step(4)
-        await self.commission_on_network(setup_code=params.commissioningParameters.setupPinCode, discriminator=params.randomDiscriminator)
+        await self.commission_on_network(setup_code=params.commissioningParameters.setupPinCode, discriminator=params.randomDiscriminator, expected_error=0x00000032)
```
please add a named constant for 0x32.

Beyond that, test plan says "This may be a failure to find the commissionable node or a failure to establish a PASE connection." ... add a comment explaining why 0x32 is the general code here? Will this throw on success?

Should we instead have a try/catch and treat all errors as ok (or at least all chiperrors as one)?

doublemis1 replied:
I can add those constant, I based on existing script approach validation

arkq commented on src/python_testing/TC_CADMIN_1_5.py:
```diff
@@ -46,15 +46,14 @@
                     filterType=ChipDeviceCtrl.DiscoveryFilterType.LONG_DISCRIMINATOR, filter=discriminator)
-            asserts.assert_true(int(cm.exception.code) == expected_error,
-                                'Unexpected error code returned from Commissioning Attempt')
+            asserts.assert_true(int(cm.exception.code) == 0, 'Unexpectedly Commissioning Attempt Passed')
```
The `with asserts.assert_raises` asserts that there was exception, this code does nothing. Also you are using `cm` beyond context manager which does not look good.

doublemis1 committed:
[CertTest][CADMIN_1_5] Fix the test case

Fix the test case to expect failures during onnetwork
commissioning in steps 4 and 7.

Signed-off-by: Michał Szablowski <michal.szablowski@nordicsemi.no>

doublemis1 committed:
Revert "Disable cadmin 1.5 from CI (#42060)"

This reverts commit 3bc876968aa9b13d9672d7ce2d608984feac421e.
